gradle.properties

```diff
@@ -18,5 +18,5 @@ ideaEdition = IC
 ideaVersion = 2017.1
 javaVersion = 1.8
 ijPluginRepoChannel = alpha
-version = 17.2.2_2017-SNAPSHOT
+version = 17.2.3
```

Comment:
yes but technically this is the 2017 branch so the next release from this branch would actually be _2017 if we happened to do it for whatever reason. If we target 2017 from master next, like you suggested, we'd be releasing from master.